galaxy/templates/configmap-nginx.yaml

```
@@ -61,7 +61,9 @@ data:
                 add_header X-Frame-Options SAMEORIGIN;
                 add_header X-Content-Type-Options nosniff;
             }
-
+            location /training-material/ {
```

Should the proxy prefix be added here?
```suggestion
            location {{ template "galaxy.add_trailing_slash" .Values.ingress.path }}training-material/ {
```

---

No, I don't believe so; I tried that first and it didn't work during my testing.  Even when using a proxy prefix the expected location is `/training-material/` and not `/galaxy/training-material/` (or whatever the proxy prefix is).

---

Hmm, so what happens if you install galaxy at a proxy prefix? The ingress wouldn't be intercepting `/` at that point anyway, so would this break?

---

That "should be" handled by adding an ingress path https://github.com/galaxyproject/galaxy-helm/blob/50e2e6f4bee6a800fa06c21958ffa05ae039faff/galaxy/values.yaml#L197

---

My last comment was incorrect, but the training hook still works with some caveats.  I suspect there would be problems if you wanted to redirect to different training-material servers for some reason, and when you enable/disable the web hook on one Galaxy instance you enable/disable it on all of them.  So [https://example.com/galaxy2]() can't disable the web hook if [https://example.com/galaxy1]() has in enabled.